release-please-config.json

```diff
@@ -12,7 +12,11 @@
             "package-name": "hugr",
             "include-component-in-tag": true,
             "draft": false,
-            "prerelease": false
+            "prerelease": false,
+            "extra-files": [{
+                "type": "generic",
```

Comment:
I think you can get rid of this line and have release-please interpret __init__ as a python file based on its extension?

Reply:
I just meant the `type` field, not the whole `extra-files` bit!

Reply:
But it seems to work... I'm a bit surpised!